src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -596,6 +599,34 @@ private static Object getSubSectionName(int subSectionPos) {
         return result;
     }
 
+    private static void validateDescriptionSection(String fileName, String sectionName,
+            Node subSection) {
+        // Till https://github.com/checkstyle/checkstyle/issues/5777
+        if ("config_filters.xml".equals(fileName) && "SuppressionXpathFilter".equals(sectionName)) {
+            validateListOfSuppressionXpathFilterIncompatibleChecks(subSection);
+        }
+    }
+
+    private static void validateListOfSuppressionXpathFilterIncompatibleChecks(Node subSection) {
+        final List<Set<String>> list = XmlUtil.findChildElementsByTag(subSection, "ul").stream()
+            .map(node -> {
+                return XmlUtil.getChildrenElements(node).stream()
+                        .map(Node::getTextContent)
+                        .collect(Collectors.toSet());
+            })
+            .collect(Collectors.toList());
+
+        assertWithMessage(
+            "Incompatible check list should match XpathRegressionTest.INCOMPATIBLE_CHECK_NAMES")
+            .that(XpathRegressionTest.INCOMPATIBLE_CHECK_NAMES)
+            .isIn(list);
```

Comment:
please remove extra list item:
![image](https://user-images.githubusercontent.com/812984/71769751-dd12d000-2ed9-11ea-91ad-39ac496811b1.png)

This assert is very hard to read ....

I do understand that you try to cut all corners to make this PR quick. I agree but please create issue to update this UT to get list of all modules, get list of modules from xpath `it`, and make sure that [all modules] - [already in xpath] = [temporary incompatible].

> [all modules]

CheckUtil.getCheckstyleChecks()

> [already in xpath]

to cut corner ..... we can hardcode it as list in XdocsPagesTest.java, to make sure Check is this list or `[temporary incompatible]`.

-----

So validation should be natural to read:
```
        assertWithMessage(
            " Check should in  XpathRegressionTest.INCOMPATIBLE_CHECK_NAMES or in XpathRegressionTest.COMPATIBLE_CHECK_NAMES")
            .that(list)
            .is(INCOMPATIBLE_CHECK_NAMES + COMPATIBLE_CHECK_NAMES);
```

Reply:
> please remove extra list item (that is not about Checks ):

The section `Description` for `SuppressionXpathFilter` contains two lists. See https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_filters.xml#L1013
I have no idea how to sort them apart. The best I could invent was to put both of them into a list and check that one of the items in that list contains the right set.

Reply:
it is internal code .... , so you can check first element to be not Check
based on `CheckUtil.getCheckstyleChecks()` and remove it from list. 
If first element is not Check remove it from consideration.

Reply:
> and make sure that [all modules] - [already in xpath] = [temporary incompatible].

This assertion already exists in `XpathRegressionTest`: See
https://github.com/checkstyle/checkstyle/pull/7406/files#diff-b6cb26ed1119a622d1bb75564fff0561R254

Reply:
In this case I completely do not understand what for such two asserts.
Please update message in asserts to help me get your ideas.

Reply:
> In this case I completely do not understand what for such two asserts.
> Please update message in asserts to help me get your ideas.
> 
> All we need is to send a message to user to put into one of list: supposed or not supported.

This two asserts to ensure that `INCOMPATIBLE_CHECK_NAMES` and `INCOMPATIBLE_JAVADOC_CHECK_NAMES` are in sync with xdocs.

Reply:
Done. Now both `ul` lists have `id` attribute.